### PR TITLE
simplify jenkins build scripts after ert reorganization

### PR DESCRIPTION
**Pull request by akva2**
Depends on https://github.com/OPM/opm-common/pull/168

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this opm-common=168 please

File: jenkins/build-pr.sh [hunk 1]
```diff
@@ -37,7 +33,7 @@ do
   fi
 done
 
-echo "Building with ert=$ERT_REVISION opm-common=$OPM_COMMON_REVISION opm-parser=${upstreamRev[opm-parser]} opm-material=${upstreamRev[opm-material]} opm-core=${upstreamRev[opm-core]} opm-grid=${upstreamRev[opm-grid]} opm-output=${upstreamRev[opm-output]} opm-upscaling=$sha1"
+echo "Building with opm-common=$OPM_COMMON_REVISION ert=${upstreamRev[ert]} opm-parser=${upstreamRev[opm-parser]} opm-material=${upstreamRev[opm-material]} opm-core=${upstreamRev[opm-core]} opm-grid=${upstreamRev[opm-grid]} opm-output=${upstreamRev[opm-output]} opm-upscaling=$sha1"
```
Comment (bska) [inline jenkins/build-pr.sh]: Tangential note: Do we want/need this on a single line or might we consider multiple lines like
```sh
cat<<EOM
Building with
  opm-common    = ${OPM_COMMON_REVISION}
  ert           = ${upstreamRev[ert]}
  opm-parser    = ${upstreamRev[opm-parser]}
  opm-material  = ${upstreamRev[opm-material]}
  opm-core      = ${upstreamRev[opm-core]}
  opm-grid      = ${upstreamRev[opm-grid]}
  opm-output    = ${upstreamRev[opm-output]}
  opm-upscaling = $sha1
EOM
```

Comment (akva2) [inline jenkins/build-pr.sh]: we can prettify. i'm busy working on MPI builds for jenkins and the way i do that changes this code anyways, so i'll include multi-line prints there.

Comment (bska) [inline jenkins/build-pr.sh]: :+1:

**Pull request merged**